Исправление ошибок

Rashad5555 opened the pull request:
#1470 Fix 1 Maintainability, 6 Style issues in LibOfTimetableOfClasses\MTeacher.cs

Rashad5555 committed:
do issue 1470

Rashad5555 committed:
solve conflict

YuriSilenok reviewed (changes requested):
изменены лишние файлы

Rashad5555 committed:
Update UnitTestOfTimetableOfClasses.csproj

#1470 Fix 1 Maintainability, 6 Style issues in LibOfTimetableOfClasses\MTeacher.cs

Rashad5555 committed:
Update packages.config

YuriSilenok commented on UnitTestOfTimetableOfClasses/UnitTestOfTimetableOfClasses.csproj:
```diff
@@ -1,6 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="15.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <Import Project="..\packages\MSTest.TestAdapter.1.4.0\build\net45\MSTest.TestAdapter.props" Condition="Exists('..\packages\MSTest.TestAdapter.1.4.0\build\net45\MSTest.TestAdapter.props')" />
+  <Import Project="..\packages\MSTest.TestAdapter.1.4.0\build\net45\MSTest.TestAdapter.props" Condition="Exists('..\packages\MSTest.TestAdapter.2.2.4\build\net45\MSTest.TestAdapter.props')" />
```
нет

YuriSilenok commented on UnitTestOfTimetableOfClasses/UnitTestOfTimetableOfClasses.csproj:
```diff
@@ -111,8 +111,8 @@
-    <Error Condition="!Exists('..\packages\MSTest.TestAdapter.1.4.0\build\net45\MSTest.TestAdapter.props')" Text="$([System.String]::Format('$(ErrorText)', '..\packages\MSTest.TestAdapter.1.4.0\build\net45\MSTest.TestAdapter.props'))" />
-    <Error Condition="!Exists('..\packages\MSTest.TestAdapter.1.4.0\build\net45\MSTest.TestAdapter.targets')" Text="$([System.String]::Format('$(ErrorText)', '..\packages\MSTest.TestAdapter.1.4.0\build\net45\MSTest.TestAdapter.targets'))" />
+    <Error Condition="!Exists('..\packages\MSTest.TestAdapter.1.4.0\build\net45\MSTest.TestAdapter.props')" Text="$([System.String]::Format('$(ErrorText)', '..\packages\MSTest.TestAdapter.2.2.4\build\net45\MSTest.TestAdapter.props'))" />
+    <Error Condition="!Exists('..\packages\MSTest.TestAdapter.1.4.0\build\net45\MSTest.TestAdapter.targets')" Text="$([System.String]::Format('$(ErrorText)', '..\packages\MSTest.TestAdapter.2.2.4\build\net45\MSTest.TestAdapter.targets'))" />
```
нет

YuriSilenok commented on UnitTestOfTimetableOfClasses/UnitTestOfTimetableOfClasses.csproj:
```diff
@@ -111,8 +111,8 @@
-  <Import Project="..\packages\MSTest.TestAdapter.1.4.0\build\net45\MSTest.TestAdapter.targets" Condition="Exists('..\packages\MSTest.TestAdapter.1.4.0\build\net45\MSTest.TestAdapter.targets')" />
-</Project>
\ No newline at end of file
+  <Import Project="..\packages\MSTest.TestAdapter.1.4.0\build\net45\MSTest.TestAdapter.targets" Condition="Exists('..\packages\MSTest.TestAdapter.2.2.4\build\net45\MSTest.TestAdapter.targets')" />
```
нетъ

YuriSilenok reviewed (changes requested):
https://gm1.ggpht.com/C_G8U4UPifApCyDerI-OHwmAmsK_dKoLwovxCyZ2N-NH6iimJlPQyHsEnqRDZOBawb8PDq3g8qUzUDK0tpdPpRpXv8lKXEjdvTB9u9AKQX4Dc8Ohl3i9f-GCjEZlR_wxS5GzZEu4GwgAAE-Fn3fX-TYjeAqwSfk05Nr-zq4xwtJ1CANaFyYREnYFFHGl_nX2IS9-Q9Y15Uz8Tv3EXjDZ0KdpAHA1ulF6ucjp7nkXr6rhPlPz4ll4XWMBQ2relAbERWGyh8DvOXUtOJwtp_hsGzXi2QaAOYaAkkxbn0IFje6SUI1z0KlXJ6b0GvgEF4qMExIkuVEUJqac-ejR0qxMjbYX18ffkJtBXAZfdZwJAyWzrYwXCJqIRxEEaWoFu2ri9nw7g3RK5uA-d1sfAH4F3MZD_ENS5QK7yftS0k9yaFOILyZ5BHYO6vNMvs_oR2G9g8zDqgNbE6GdpZVaLJwz_M-RlxQuT7zm-gubyCEg_myAiySOCBPz7DWFShEN8Pmd4ooL_LtH1wNJdgmt6uJ0p0HPgbQ92NQts6DsH8o8vnJ5jCNZ4bj1hMTyeGwylm89P30VaXCkgej0_t6Yby4Vm2QXC1GUTjFrE9gHaqmqH-xZpLpwIYh3xrwyBjnHL-Jj1MQ1HtDbagKFXoOlyHZ61jLh3gvV_oG32z1LZCeCuTX9sW0-gCjvd5UTTyWcBsGki3Z6GvG7G4bDG-f9EkysYQUpRAhNrrNGbYY0KO9ZU8R2HFD-irQfZt68ZVKe7XWf-k8s=s0-l75-ft-l75-ft

Rashad5555 committed:
Update UnitTestOfTimetableOfClasses.csproj

Rashad5555 committed:
Update UnitTestOfTimetableOfClasses.csproj
